custom/icds_reports/models/aggregate.py

```diff
@@ -347,6 +349,51 @@ class ChildHealthMonthly(models.Model, AggregateMixin):
     lunch_count = models.IntegerField(blank=True, null=True)
     state_id = models.TextField(blank=True, null=True)
     opened_on = models.DateField(blank=True, null=True)
+    birth_weight = models.PositiveSmallIntegerField(null=True, help_text="birth weight in grams")
+    mother_id = models.TextField(blank=True, null=True)
```

Comment:
This name is very misleading. The actual property this refers to is the person case of the child, not the mother. If that is intentional, and you want the child person id here, it would be way easier to understand if it were named as such. maybe something like `person_id`

Reply:
Where will this be used?